test/NuGet.Core.FuncTests/Dotnet.Integration.Test/DotnetListPackageTests.cs

```diff
@@ -473,13 +478,13 @@ public async Task ListPackage_WithHttpSource_Warns()
 
             using var mockServer = new FileSystemBackedV3MockServer(pathContext.PackageSource);
             mockServer.Start();
-            pathContext.Settings.AddSource("http-source", mockServer.ServiceIndexUri);
+                pathContext.Settings.AddSource("http-source", mockServer.ServiceIndexUri);
```

Comment:
is the whitespace correct?

Reply:
🤦  https://developercommunity.visualstudio.com/t/Make-it-more-obvious-when-compare-with/10418789